### ISSUE TITLE
25427812: Directory Services indicator dot steals keyboard focus at login window.

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#7876]
#### Description

The colored Directory Services status indicator dot will show up on the login screen when a connection is not available. This dot appears some time after the rest of the login window UI is displayed. This dot is positioned between the 'user name' and 'password' fields, shows up at an unpredictable time, takes keyboard focus, and Full Keyboard Access always defaults to enabled on the login window. As such, the user cannot predict how many 'tab' key presses it will take to select the Password field.
## 

Product Version: 10.11.4
Created: 2016-03-29T23:35:15.817180
Originated: 2016-03-29T00:00:00
Open Radar Link: http://www.openradar.me/25427812

**Comment by openradar-mirror** [comment]
Modified: 2016-03-30T20:35:23.006310
